client/src/sass/include/_mixins.scss

```diff
@@ -225,7 +225,7 @@
   line-height: $button-height;
   border-radius: 3px;
   text-align: center;
-  padding: 0 17px 0 13px;
```

Comment:
Is it a mistake? If no, should be discussed in another PR

Reply:
I noticed a small regression, this proved to be a quickfix but you're right, it shouldn't be in this PR, sorry.